cmd/sf/wikidata_test.go

```diff
@@ -61,6 +61,7 @@ func setupWikidata(pronomx bool) error {
 // identificationTests provides our structure for table driven tests.
 type identificationTests struct {
 	fname          string
+	label          string
```

Comment:
Label added to testing structures to assert the output in our different encodings works as anticipated. Struct data updated below.